packages/playwright/types/test.d.ts

```diff
@@ -8596,6 +8596,11 @@ interface TestProject {
        * timeout for toPass method in milliseconds.
        */
       timeout?: number;
+
+      /**
+       * interval for toPass method in milliseconds.
+       */
+      intervals?: number[];
```

Comment:
This file is generated from docs/src/test-api/class-testconfig.md, you should edit the .md file and then run build locally to update this file.